src/release/breaking-changes/describe-enum.md

```diff
@@ -1,40 +1,51 @@
 ---
-title: Migration guide for describeEnum
+title: Migration guide for describeEnum and EnumProperty
 description: Removal of describeEnum and how to migrate
 ---
 
 ## Summary
 
-The global method `describeEnum` has been deprecated. Existing uses
+The global method `describeEnum` has been deprecated. Previous uses
 of `describeEnum(Enum.something)` should use
 `Enum.something.name` instead.
 
+The class EnumProperty was modified to extend `<T extends Enum?>` instead of `<T>`. Existing uses of `EnumProperty<NotAnEnum>` should use `DiagnosticsProperty<NotAnEnum>` instead.
+
 ## Context
 
-Dart 2.17 introduced enhanced enums. With them, Enum became a type
-and all enums got a `name` getter, which made `describeEnum` redundant.
+Dart 2.17 introduced [enhanced enums][], which added `Enum` as a type. As a result, all enums got a `name` getter, which made `describeEnum` redundant. Before that, enum classes were often diagnosticated using an `EnumProperty`.
```

Comment:
I'm so tempted to leave diagnosticated in, but we _should_ replace it something like, tested, evaluated, analyzed, investigated...

```suggestion
Dart 2.17 introduced [enhanced enums][], which added `Enum` as a type.
As a result, all enums got a `name` getter, which made `describeEnum`
redundant. Before that, enum classes were often
analyzed using an `EnumProperty`.
```